src/app/workflow/dag/dag.component.html

```diff
@@ -70,6 +78,9 @@
         [hidden]="dagType !== 'cwlviewer'" [workflow]="(workflow$ | async)" [selectedVersion]="selectedVersion"
         [refresh]="refreshCounter" [expanded]="expanded">
       </app-cwl-viewer>
+      <app-wdl-viewer *ngIf="(isWDL$ | async) && (descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL && (workflow$ | async)?.source_control_provider === 'GITHUB' && dagType === 'wdlviewer'"
```

Comment:
Same as earlier comment -- isn't the second part of your and condition redundant?